installer/scripts/omsadmin.sh

```diff
@@ -161,6 +161,9 @@ usage()
     echo
```

Comment:
I think it would be clearer if the added echo lines were here, with the rest of the echo lines about the process limit